features/bootstrap/FeatureContext.php

```diff
@@ -83,4 +83,16 @@ public function assertPageAddressAfterAllRedirection(string $url, int $ttl = 3,
             $this->assertPageAddressAfterAllRedirection($url, $ttl, ++$try);
         }
     }
+
+    /**
+     * @When I click the :cssElementSelector
```

Comment:
What about adding more precision like :
 ```php
I click the :cssElementSelector selector
```

We will have two clearly distinct cases : 

`I click the "toto" selector` for CSS selectors
`I click the "toto" element` for ID elements

Reply:
yep good idea :+1: